### PR TITLE
feat(profile): redesign

### PR DESCRIPTION
**Pull request by olros**
Lint, test and build
## Description

Comments/issues/screenshots:
- Moved all tabs and buttons on the profile to a list which is on the left on desktop and centered on mobile. The move of buttons to the list can be discussed. I'm not sure if they should stay where they were or be moved to the list.
- The list makes the profile more flexible in terms of room for adding more content like e.g. badges.

image
image
image
image
image
image

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] The PR includes a `closes #issueID`
- [x] The PR includes a picture showing visual changes

### CODE REVIEW

File: src/components/navigation/Navigation.js [hunk 1]
```diff
@@ -269,7 +269,7 @@ function Navigation(props) {
         <title>TIHLDE</title>
       </Helmet>
       <AppBar
-        className={classNames(classes.root, fancyNavbar && scrollLength < 20 && classes.rootLanding)}
+        className={classNames(classes.root, fancyNavbar && scrollLength < 20 && !showSidebar && classes.rootLanding)}
```
Comment (olros) [inline src/components/navigation/Navigation.js:272]: This fixes an issue with the navbar being transparent when using the fancynavbar and expanding the sidepanel while on top of the page

**Before:**
image

**Now:**
image

**Review by SveinJH — APPROVED** [review]

File: src/containers/Profile/components/ProfilePaper.js [hunk 2]
```diff
@@ -143,69 +154,94 @@ function ProfilePaper(props) {
...
+    badge: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    onClick: PropTypes.func,
+  };
+
+  return (
+    <>
+      <Paper className={classes.paper} noPadding>
+        {showModal && userData && (
+          <Modal className={classes.memberProof} onClose={() => setShowModal(false)} open={showModal}>
+            <QRCode size={280} value={userData.user_id ? userData.user_id : 'hhhhh'} />
```
Comment (Zenjjim) [inline src/containers/Profile/components/ProfilePaper.js:185]: userData.user_id || 'hhhhh'

File: src/containers/Profile/components/ProfilePaper.js [hunk 3]
```diff
@@ -102,16 +86,43 @@ const styles = (theme) => ({
...
   const { classes, logoutMethod } = props;
-  const [tab, setTab] = useState(0);
   const [userData, setUserData] = useState({});
   const [isGroupMember, setIsGroupMember] = useState(false);
   const [isLoading, setIsLoading] = useState(true);
+  const eventTab = { label: 'Arrangementer', icon: EventIcon };
+  const notificationsTab = { label: 'Varsler', icon: NotificationsIcon, badge: userData.unread_notifications };
+  const settingsTab = { label: 'Profil', icon: ProfileIcon };
+  const adminTab = { label: 'Admin', icon: AdminIcon, iconEnd: NewTabIcon, component: Link, to: URLS.admin };
+  const logoutTab = { label: 'Logg ut', icon: LogOutIcon, onClick: logoutMethod };
```
Comment (Zenjjim) [inline src/containers/Profile/components/ProfilePaper.js:122]: Nå er jo ikke dette "tabs" lengre.  Kunne hatt bedre navn

File: src/containers/Profile/components/ProfilePaper.js [hunk 4]
```diff
@@ -102,16 +86,43 @@ const styles = (theme) => ({
...
+  const [tab, setTab] = useState(eventTab.label);
   const [showModal, setShowModal] = useState(false);
+  const tabs = [eventTab, notificationsTab, settingsTab, ...(isGroupMember ? [adminTab] : []), logoutTab];
```
Comment (Zenjjim) [inline src/containers/Profile/components/ProfilePaper.js:125]: Denne burde være rett etter der du setter alle tabsene

**Pull request merged**